bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/internal/model/commands/GetCleanFilterIndicatorCommand.java

```diff
@@ -30,14 +28,13 @@
 @NonNullByDefault
 public class GetCleanFilterIndicatorCommand extends BRC1HCommand {
 
-    private final Logger logger = LoggerFactory.getLogger(GetCleanFilterIndicatorCommand.class);
-
     private @Nullable Boolean cleanFilterIndicator;
 
     @Override
     public void handleResponse(Executor executor, ResponseListener listener, MadokaMessage mm)
             throws MadokaParsingException {
-
+        // In similar class GetVersionCommand.java is a note about intentionally leaving the NPE, might be intentionally
+        // here too
```

Comment:
What is the argument to leave a potential NPE ???

Reply:
In the linked file there is this comment from the developer who build the binding:
```
        // In this method, we intentionally do not check for null values in mv45 and mv46. In case of null pointer
        // access, it will be catched by the global exception and be reported as a Parsing Reponse exception.
```

I see this pattern on all `public void handleResponse` methods in this binding. Only at  the other place there are no comments, so i added them to clarify why they are left behind. I could catch them and throw a `MadokaParsingException` but the dev did this on purpose and it was merged so. 

Let me know if it needs to be changed, otherwise merge as is.

Reply:
I refactored those cases to do proper null check and throw a `MadokaParsingException` if null.